api/sys_utilization_report.go

```diff
@@ -12,15 +12,23 @@ import (
 )
 
 func (c *Sys) UtilizationReport() (*UtilizationReportOutput, error) {
-	return c.UtilizationReportWithContext(context.Background())
+	return c.UtilizationReportWithContext(context.Background(), "")
 }
 
-func (c *Sys) UtilizationReportWithContext(ctx context.Context) (*UtilizationReportOutput, error) {
+func (c *Sys) UtilizationReportWithNamespace(nsPath string) (*UtilizationReportOutput, error) {
```

Comment:
Broadly speaking, this looks good, but we should update the client in lockstep with the changes we make to support the, to avoid having an API package (even temporarily) that lacks compatibility with the API itself.